voxblox_ros/include/voxblox_ros/tsdf_server.h

```diff
@@ -95,7 +95,7 @@ class TsdfServer {
   void updateMeshEvent(const ros::TimerEvent& event);
   void publishMapEvent(const ros::TimerEvent& event);
 
-  std::shared_ptr<TsdfMap> getTsdfMapPtr() { return tsdf_map_; }
+  std::shared_ptr<TsdfMap> getTsdfMapPtr() const { return tsdf_map_; }
```

Comment:
if you need const access to the tsdf layer, wouldn't it make more sense to add a function to get a const reference? I think that would be the correct way to do it, or at the very least this should return a const shared ptr. Basically similar [how it is here in the layer to get blocks.](https://github.com/ethz-asl/voxblox/blob/master/voxblox/include/voxblox/core/layer.h#L104)